openassessment/fileupload/tests/test_file_management.py

```diff
@@ -0,0 +1,168 @@
+import json
+import mock
+
+from django.db import IntegrityError
+from django.test import TestCase
+from django.test.utils import override_settings
+from moto import mock_s3
+
+from openassessment.assessment.models.base import SharedFileUpload
+from openassessment.fileupload.api import get_student_file_key, FileUpload, FileUploadManager
+
+
+class MockBlock(object):
+
+    STUDENT_ID = 'student-id-'
+    COURSE_ID = 'course-v1:edx+ORA203+course-'
+    ITEM_ID = 'itemitemitemitemitem'
+
+    def __init__(self, number, team_id=None, descriptions=None, sizes=None, names=None, max_items=25):
+        self.student_id = MockBlock.STUDENT_ID + str(number)
+        self.course_id = MockBlock.COURSE_ID + str(number)
+        self.item_id = MockBlock.ITEM_ID + str(number)
+
+        empty_list = '[]'
+        self.saved_files_descriptions = json.dumps(descriptions) if descriptions else empty_list
+        self.saved_files_names = json.dumps(names) if names else empty_list
+        self.saved_files_sizes = json.dumps(sizes) if sizes else empty_list
+        self.MAX_FILES_COUNT = max_items
+        if team_id:
+            self.team = mock.MagicMock(
+                team_id=team_id,
+                course_id=self.COURSE_ID
+            )
+        else:
+            self.team = None
+
+    def is_team_assignment(self):
+        return bool(self.team)
+
+    def get_anonymous_user_id_from_xmodule_runtime(self):
+        return self.student_id
+
+    def get_xblock_id(self):
+        return self.item_id
+
+    def get_student_item_dict(self):
+        return dict(
+            student_id=self.student_id,
+            item_id=self.item_id,
+            course_id=self.course_id,
+            item_type='openassessment'
+        )
+
+
+def uploadDict(name, desc, size):
```

Comment:
you have strangely transitioned to camel case here and in some parameters below.